provision/acc_provision/acc_provision.py

```diff
@@ -540,11 +540,15 @@ def Raise(exception):
              else Raise(Exception("Invalid value: %s; "
                                   "Expected one of: {%s}" %
                                   (x, ','.join(y)))))))
+    isname = lambda x: (1 < len(x) < 32) & \
```

Comment:
did you really mean to use "and" instead of "&"? Also the exception raised should be say that the name is invalid instead of saying option is missing. Something like this

lambda x: (
   True if x and len(x) < 32 and x[0].isalpha() and x.replace('_', '').isalnum()
   else Raise(Exception("Invalid name"))

Reply:
Fixed that.